serverless/src/submitHarmonyOrder/__tests__/constructOrderPayload.test.js

```diff
@@ -693,4 +693,115 @@ describe('constructOrderPayload', () => {
       })
     })
   })
+
+  describe('when only sending concatenate is true', () => {
+    test('constructs a payload containing supportsConcatenation = true and enableConcatenateDownload = true', async () => {
```

Comment:
These test names are hard to follow, the payload does not contain supportsConcatenation or enableConcatenateDownload, it just contains concatenate=true

Reply:
Could you check again and see that it's what you think it should be?

Reply:
```
      expect(response.getAll('concatenate')).toEqual([
        'true'
      ])
```

You aren't testing for either supportsConcatenation or enableConcatenateDownload, so the test name shouldn't say you are testing for them.

describe: describe the conditions for the test - 'when supportsConcatenation is true and enableConcatenateDownload is true'
test: describe the desired outcome of the test - 'constructs a payload with concatenate true'

All of the added tests in this file should follow that formula

Reply:
I see. I'll change the tests to match that convention.